modules/lvms-installing-logical-volume-manager-operator-disconnected-environment.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_content-type: PROCEDURE
+[id="lvms-installing-lvms-disconnected-env_{context}"]
+= Installing {lvms} in a disconnected environment
+
+You can install {lvms} on {product-title} {product-version} in a disconnected environment by following the procedures in the _About disconnected installation mirroring_ section.
+
+.Prerequisites
+
+* You read the _About disconnected installation mirroring_ section.
+* You have access to the {product-title} image repository.
+* You created a mirror registry.
+
+.Procedure
+
+. Follow the steps in the _Creating the image set configuration_ procedure. To create an `ImageSetConfiguration` for {lvms}, you can use the following example YAML file:
++
+include::snippets/lvms-disconnected-ImageSetConfig.adoc[]
+
+. Follow the procedure in the _Mirroring an image set to a mirror registry_ section.
+
+. Follow the procedure in the _Installing the ImageContentSourcePolicy and CatalogSource resources into the cluster_ section.
```

Comment:
FYI: If you aren't aware, [ICSPs are deprecated as of 4.13](https://github.com/openshift/openshift-docs/pull/56250/files). They are still in the code, but the plan is to remove them at some point. 
There are two new objects to [configure mirroring](https://docs.openshift.com/container-platform/4.13/openshift_images/image-configuration.html#images-configuration-registry-mirror_image-configuration).

Reply:
@radeore Could you please take another look at the docs? To limit future updates, I made the change of swapping out the `Installing the ImageContentSourcePolicy and CatalogSource resources into the cluster` section to `Configuring image registry repository mirroring` as per the above comment.